versions.yaml

```diff
@@ -7,3 +7,4 @@ jsonnetfmt: 0.16.0
 goimports: 0.1.0
 delve: 1.6.0
 gotestsum: 1.7.0
+lintroller: george-e-shaw-iv/feat/tier-list
```

Comment:
This needs to be updated before merge I assume

Reply:
```suggestion
lintroller: 1.6.0
```